images/macos/templates/macOS-11.pkr.hcl

```diff
@@ -0,0 +1,242 @@
+variable "source_vm_name" {
+  type = string 
+  default = "clean_macos_11_300gb"
+}
+
+variable "source_vm_tag" {
```

Comment:
What does this variable mean? 
Also, I think typo in default: `bigsur_300g` -> `bigsur_300gb`